Sources/SwiftBundler/Extensions/Process.swift

```diff
@@ -7,12 +7,12 @@ import Foundation
 /// All processes that have been created using `Process.create(_:arguments:directory:pipe:)`.
 ///
 /// If the program is killed, all processes in this array are terminated before the program exits.
-var processes: [Process] = []
+public var processes: [Process] = []
```

Comment:
`processes` is now a free, public variable "polluting" the library user namespace. I'd prefer to encapsulate this in a enum e.g. `Process`. What do you think @stackotter?

Reply:
Yeah I reckon namespacing it as a static variable would make sense. I'll probably try finishing the transition away from `Process` soon, which will involve creating a new type to namespace all the new process related methods. It'd make sense for it to be attached to that type when it happens.

Maybe for now they can stay global but internal, and static methods can be added to `Process` to house the shutdown logic currently housed in the CLI main function.

Reply:
I've decided to go a different route and just add a public function `Process.killAllRunningProcessesOnExit()`. This gets rid of the need to expose any of this and refactoring can be done whenever. 
Changed with 216e1a8cb4a1442477c43ce4db7b51ed6883cf78